pages/04.HDM Doc 2.2/07.vcd/80.Troubleshooting Guide/default.md

```diff
@@ -55,19 +55,14 @@ CP-5107: On HDM_Cloud_Cache reboot, the cache service does not come up. An [HDM
 
 CP-5112: If the on-premises VM remains powered on following a cold or warm migration, any virtual machine configured to sync data via a static IP address will cause an IP address collision upon power-on. As a result, the migrated VM will be unable to employ the newly-configured static IP address. This is mainly seen in the SLES Linux distribution where IP address collisions are detected as part of the “network startup” scripts, and IP addresses fail to come online. However, regardless of the specific Linux or Windows distribution, any time two VMs are assigned the same IP address, only one will be reachable through that IP address. To avoid this issue, only keep one VM (either the migrated or the original on-premises) powered on at any given time.
 
+DP-2957: Commit operation fails for Linux VMs. This is because the connection between PIO Manager and HA Manager had failed.
+
```

Comment:
I think, the commit issue is not for all the linux vms. This statement is very generic and user will get confused. So we can may remove this.

Reply:
Resolved..